elemnets/textInput.js

```diff
@@ -10,6 +11,15 @@ class WebTextInput extends WebElement{
         console.log('Clearing field');
         await this.protractorElement.clear();
     }
+
+    async check() {
+        console.log('Checking checkbox');
+        let checked = await this.protractorElement.isSelected();
+        if(!checked){
+            await this.protractorElement.click();
+        }
+    }
+
```

Comment:
Create a second method `uncheck`, or handle unchecking possibility in previous method

Reply:
Done